svg/global_attributes.json

```
@@ -1400,7 +1400,10 @@
       "font-stretch": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Reference/Attribute/font-stretch",
-          "spec_url": "https://drafts.csswg.org/css-fonts/#font-stretch-prop",
+          "spec_url": [
+            "https://drafts.csswg.org/css-fonts/#font-stretch-prop",
+            "https://svgwg.org/svg2-draft/styling.html#PresentationAttributes"
```

I think a text fragment would be more helpful here:

```suggestion
            "https://svgwg.org/svg2-draft/styling.html#:~:text=font-stretch"
```